### PR TITLE
refactor(Form & FormControl): refactor css variables

### LINKED ISSUE #991: Use a configuration first method (modifiers and states) for Form
Relates directly back to #273

### PR DESCRIPTION
**Pull request by christiemolloy**
closes #991 and #992

### CODE REVIEW

File: src/patternfly/components/Form/form.scss [hunk 1]
```diff
@@ -10,8 +10,7 @@
   --pf-c-form__label--FontWeight: var(--pf-global--FontWeight--normal);
   --pf-c-form__label--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-form__label--LineHeight: var(--pf-global--LineHeight--sm);
-  --pf-c-form__label--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-form__label--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__label--PaddingY: var(--pf-global--spacer--sm);
```
Comment (mcoker) [inline src/patternfly/components/Form/form.scss]: Looks like you need to update https://github.com/patternfly/patternfly-next/blob/c4931220152058f1a0a55bd8adf02d6ebb8ab024/src/patternfly/components/Form/form.scss#L66 and https://github.com/patternfly/patternfly-next/blob/c4931220152058f1a0a55bd8adf02d6ebb8ab024/src/patternfly/components/Form/form.scss#L86

**Comment by mcoker** [comment]
Could this line https://github.com/patternfly/patternfly-next/blob/c4931220152058f1a0a55bd8adf02d6ebb8ab024/src/patternfly/components/FormControl/form-control.scss#L93 redefine `--pf-c-form-control--BorderBottomColor`?

**Comment by mcoker** [comment]
same as ^^ for this line? https://github.com/patternfly/patternfly-next/blob/c4931220152058f1a0a55bd8adf02d6ebb8ab024/src/patternfly/components/FormControl/form-control.scss#L134

**Comment by mcoker** [comment]
@matthewcarleton @srambach this should be ok to review now.

**Comment by srambach** [comment]
I'm not sure if this was working before, but if these components are placed in a dark container, some are illegible (danger input, danger textarea, and all the selects).

**Pull request merged**